woocommerce/woocommerce
# Add/a2p create customers crud api core tests

### All Submissions:






### Changes proposed in this Pull Request:
Adding in customers api tests into our api-core-test suite to obtain more test coverage



Closes 360-gh-woocommerce/woocommerce-quality

### How to test the changes in this Pull Request:

1. Pull down the code
2. navigate to `plugins/woocommerce/tests/api-core-tests'
3. execute USE_WP_ENV=1 pnpm playwright test --config=tests/api-core-tests/playwright.config.js
4. all `api-core-tests` (including the newly added customer tests) should execute successfully

### Other information:




### FOR PR REVIEWER ONLY:
